state-chain/pallets/cf-broadcast/src/tests.rs

```diff
@@ -996,6 +997,7 @@ fn aborted_broadcasts_can_still_succeed() {
 				crate::Event::<Test, Instance1>::BroadcastSuccess {
 					broadcast_id,
 					transaction_out_id,
+					transaction_hash: Default::default(),
```

Comment:
it's not clear why this is Default::default() - should be some association we can make in the test?